manifests/i/IvanCharapanau/Harbor/0.3.0/IvanCharapanau.Harbor.installer.yaml

```
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 Dumplings Mod
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: IvanCharapanau.Harbor
+PackageVersion: 0.3.0
+UpgradeBehavior: install
+ReleaseDate: 2025-03-01
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  Scope: user
+  InstallerUrl: https://github.com/av/harbor/releases/download/v0.3.0/Harbor_0.3.0_x64-setup.exe
+  InstallerSha256: D44E53DDBBB7AB6473C4746D23432015294112640A4840341A1E0E05721E3185

```

```suggestion
  InstallerSha256: E964C33BF7E77E5C4CA92BF1EED085CD084A2477D2F4B92C51194683B3D71DA8
```

(Automated response - build 988.)